pkgs/applications/audio/reaper/default.nix

```
@@ -8,18 +8,20 @@
 , lame
 , ffmpeg
 , vlc
+, xdg-utils
+, which
 
 , jackSupport ? true, libjack2
 , pulseaudioSupport ? config.pulseaudio or true, libpulseaudio
 }:
 
 stdenv.mkDerivation rec {
   pname = "reaper";
-  version = "6.25";
+  version = "6.27";
 
   src = fetchurl {
     url = "https://www.reaper.fm/files/${lib.versions.major version}.x/reaper${builtins.replaceStrings ["."] [""] version}_linux_x86_64.tar.xz";
-    sha256 = "0i1idlr4ar28wvwcvwn9hqzb63kki1x1995cr87a9slxfa7zcshb";
+    hash = "sha256-6zHZK1ZYVSuCwZFWiBNJWhlk7Sau/paw+FMAaiVJHbg=";
   };
 
   nativeBuildInputs = [ autoPatchelfHook makeWrapper ];
```

```suggestion
  nativeBuildInputs = [
    autoPatchelfHook
    makeWrapper
    xdg-utils # Required for desktop integration
    which
  ];
```

they are used in install-reaper.sh.

---

Fixed. Thanks.